docs/workforce-management-framework/topics/pro-using-demo-app.adoc

```diff
@@ -0,0 +1,110 @@
+= Using the demo application user interface
+
+The {WFM-RC-LongName} demo application consists of three components:
+
+- A mobile application for end users
+- A portal application for administrator users on a web browser
+- A cloud application for backend server functionality (No user interface)
+
+The demo application has demo users and demo data to show off the functionality of {WFM-RC-LongName}.
+The demo application can define business process or procedure via a custome Workflow and assign these workflows as jobs to
+individual workers as workorders. Workers can view their workorder via a mobile applicaion and administrators
+can managed workorders via the web portal.
+
+== Mobile application
```

Comment:
Why mobile application is described first. 
That's not how workflow works. 

I can do the quick demo for you to show how this should be represented. 
Basically it's:
Portal:
- Create workflow
- Create workorder
Finish workorder
Do not present here any steps itself as they may differ. 
Just mention that steps can come from workflow and be developed.